### PR TITLE
Add storybook as separate lib project

### PR DESCRIPTION
**Pull request by tjeerddie**

### CODE REVIEW

File: libs/orchestrator-ui-components-storybook/public/logo192.png
Comment (ricardovdheijden) [inline libs/orchestrator-ui-components-storybook/public/logo192.png:1]: Is the public folder really needed?

File: libs/orchestrator-ui-components-storybook/package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+    "name": "storybook-app",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@elastic/datemath": "^5.0.3",
+        "@elastic/eui": "^81.0.0",
+        "@emotion/css": "^11.11.0",
+        "@emotion/react": "^11.11.0",
+        "@orchestrator-ui/orchestrator-ui-components": "^0.0.1-alpha.9",
```
Comment (ricardovdheijden) [inline libs/orchestrator-ui-components-storybook/package.json:10]: Is there a way to use the actual current version of the lib? The currect setup requires a publish to NPM before implementing the Storybook parts.

File: libs/orchestrator-ui-components-storybook/src/components/Table/DataGridTable.stories.tsx [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
 
-const extractedArr = (arr, start, howMany) =>
-    arr.filter((_, index) => {
+const extractedArr = (arr: Array<any>, start: number, howMany: number) =>
+    arr.filter((_: any, index: number) => {
         return index >= start && index < howMany + start;
     });
 
-const TableWithEvents = (args) => {
+const TableWithEvents = (args: any) => {
```
Comment (ricardovdheijden) [inline libs/orchestrator-ui-components-storybook/src/components/Table/DataGridTable.stories.tsx:14]: Isn't the type of args just "DataGridTableProps", that can be imported from the DataGridTable component (this remark counts for all similar story book cases)

File: libs/orchestrator-ui-components-storybook/public/logo192.png
Comment (tjeerddie) [reply libs/orchestrator-ui-components-storybook/public/logo192.png:1]: public is needed for storybook

**Comment by ricardovdheijden** [comment]
Functionality implemented through the NX upgrade v16

**Pull request closed without merging**